### DISCUSSION TITLE
怎么把英语输入方案导出给其他终端的rime客户端上使用

### DISCUSSION THREAD

**Discussion by um-nyu** [fcitx5-android/fcitx5-android#422] [Q&A | 问答]
![20240119_225137](https://github.com/fcitx5-android/fcitx5-android/assets/81240333/c4bc9b5f-21e7-4ae5-9a1c-1f959a1c5453)
 如图

**Comment by rocka** [comment]
英文补全功能来自 fcitx5 的 spell 模块 https://github.com/fcitx/fcitx5/tree/5.1.6/src/modules/spell ，跟 rime 没有关系。
